Resources/Prototypes/_Sunrise/Entities/Clothing/Neck/cloaks.yml

```
@@ -216,6 +216,15 @@
   - type: Sprite
     sprite: _Sunrise/Clothing/Neck/Cloaks/atmosian.rsi
 
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingFairyWings
+  name: Fairy wings
+  description: The wings of the legendary fairy.
+  components:
+  - type: Sprite
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)